requirements-sync.txt

```diff
@@ -0,0 +1,79 @@
+#
```

Comment:
It took me a second to piece together what is going on here. Would you please update README.md about installing using this file to run the sync tool as this is a little non-standard? Maybe add a comment on how this repository contains both a library and a tool.

Reply:
So there is a description of this process the [readme](https://github.com/NCI-GDC/gdc-models?tab=readme-ov-file#update-the-data-models) and there are more docs for the models [here](https://gdc-ctds.atlassian.net/wiki/spaces/GDC/pages/347570178/GDC+Models). I've updated the readme with the updated installation cmds but otherwise have left it as is. If you think further docs are needed, let me know or feel free to extend the confluence as well.